specs/AgriFood/AgriFarm/example-normalized-ld.jsonld

```diff
@@ -0,0 +1,71 @@
+{
+    "@context": [
+        "http://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
+        "https://schema.lab.fiware.org/ld/jsonldcontext.jsonld"
+    ],
+    "id": "urn:ngsi-ld:AgriFarm:72d9fb43-53f8-4ec8-a33c-fa931360259a",
+    "type": "AgriFarm",
+    "createdAt": "2017-01-01T01:20:00Z",
+    "modifiedAt": "2017-05-04T12:30:00Z",
+    "name": {
+        "type": "Property",
+        "value": "Wheat farm"
+    },
+    "description": {
+        "type": "Property",
+        "value": "A farm producing wheat"
+    },
+    "location": {
+        "type": "GeoProperty",
+        "value": {
+            "type": "Point",
+            "coordinates": [[100,0], [101,0]]
+        }
+    },
+    "landLocation": {
+        "type": "GeoProperty",
+        "value": {
+            "type": "Polygon",
+            "coordinates": [[
+                [100,0],
+                [101,0],
+                [101,1],
+                [100,1],
+                [100,0]
+            ]]
+        }
+    },
+    "address": {
+        "type": "Property",
+        "value": {
+            "addressLocality": "Valdepeñas",
+            "addressCountry": "ES",
+            "streetAddress": "Camino de Membrilla 17"
+        }
+    },
+    "contactPoint": {
```

Comment:
please use schema.org https://schema.org/ContactPoint

Reply:
do you mean including the type using the `@type`?

I mean:

```
"contactPoint": {
  "type": "Property",
  "value": {
    "@type": "ContactPoint",
    "@value": {
            "email": "wheatfarm@email.com",
            "telephone": "00349674532"
        }
  }
}
```

Reply:
good point. it would be simpler 

````
"contactPoint": {
  "type": "Property",
  "value": {
    "type": "ContactPoint",
    "email": "wheatfarm@email.com",
     "telephone": "00349674532"
  }
}
````
(it could be also @type as type is an alias of @type in NGSI-LD)

the @type, @value pattern is for data typing not for node (object structure) typing